src/pip/_internal/vcs/bazaar.py

```diff
@@ -5,8 +5,7 @@
 
 import logging
 import os
-
-from pip._vendor.six.moves.urllib import parse as urllib_parse
+from urllib import parse as urllib_parse
```

Comment:
As a general follow up note, that I'll be happy to put into an issue: we should change these `urllib_*` imports into usages that look like `urllib.*`

Reply:
IOW, Change `from urllib import xyz as urllib_xyz` to `import urllib.xyz` and the correct usage further down. L